### PR TITLE
Add efficientnetb0 model

### PR DESCRIPTION
**Pull request by sukyoungjeong-furiosa**
## Changes

- Deploy an initial model implementation for efficienent-b0, source from [rwightman/gen-efficientnet-pytorch](https://github.com/rwightman/gen-efficientnet-pytorch).
- Exported the pytorch model to onnx as follows:
```
import torch
import onnx
from tempfile import NamedTemporaryFile
import furiosa.optimizer.frontend.onnx

torch_model = torch.hub.load(
    "rwightman/gen-efficientnet-pytorch", "efficientnet_b0", pretrained=True, trust_repo=True
)
torch_model = torch_model.eval()
with NamedTemporaryFile(suffix=".onnx") as file:
    torch.onnx.export(
        torch_model, torch.randn(1, 3, 224, 224), file, opset_version=13
    )
    onnx_model = onnx.load_model(file.name)

onnx.checker.check_model(onnx_model, full_check=True)
optimized_onnx_model = furiosa.optimizer.frontend.onnx.optimize_model(onnx_model)
onnx.checker.check_model(optimized_onnx_model, full_check=True)
optimized_onnx_model = optimized_onnx_model.SerializeToString()
```
- Accuracy: 16.104% (original f32 model: 77.698%)

### CODE REVIEW

**Comment by furiosa-infra** [comment]
0dc3ceb Minor fix in regression test
```

--------------------------------------------------------- benchmark: 1 tests --------------------------------------------------------
Name (time in ms)                             Min       Max     Mean  StdDev   Median     IQR   Outliers      OPS  Rounds  Iterations
-------------------------------------------------------------------------------------------------------------------------------------
test_mlcommons_efficientnetb0_accuracy     7.7981  352.8378  18.7849  8.9386  17.8206  3.2171  1439;3194  53.2341   50000           1
-------------------------------------------------------------------------------------------------------------------------------------

Legend:
  Outliers: 1 Standard Deviation from Mean; 1.5 IQR (InterQuartile Range) from 1st Quartile and 3rd Quartile.
  OPS: Operations Per Second, computed as 1 / Mean
```

**Review by furiosamg — APPROVED** [review]
Looks good to me overall. I left a small minor comment. Thanks!

File: furiosa/models/vision/efficientnet_b0/__init__.py [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+class EfficientNetB0(ImageClassificationModel):
+    """EfficientNet B0 model"""
+
+    @staticmethod
+    def get_artifact_name():
+        return "efficientnet_b0"
+
+    @classmethod
+    def load_aux(cls, artifacts: Dict[str, bytes], use_native: bool = True, *args, **kwargs):
```
[the remark this answers is not in the document]
Comment (sukyoungjeong-furiosa) [reply furiosa/models/vision/efficientnet_b0/__init__.py:87]: I didn't quite get the comment. Actually, there's no intention at setting `use_native` as default `True`, which does nothing in this context. Would there be a better way to indicate that the value is never used?

[the remark this answers is not in the document]
Comment (furiosamg) [reply furiosa/models/vision/efficientnet_b0/__init__.py]: Resolved by offline chat. Thanks
